engine/swarm/admin_guide.md

```diff
@@ -53,6 +53,29 @@ updates to the swarm. A quorum of managers must also agree on node additions
 and removals. Membership operations are subject to the same constraints as state
 replication.
 
+### Maintaining a quorum of managers
+
+Always maintain **an odd number of manager nodes** in the swarm. Otherwise, your
+swarm may lose a quorum of managers, and will not be able to perform management
```

Comment:
Having an even number of managers doesn't cause you to lose quorum, it just doesn't have any fault tolerance benefit over having one fewer manager. This is described in detail in the "Add manager nodes for fault tolerance" section below.